json/openwrt.json

```diff
@@ -20,7 +20,7 @@
             "resources": {
                 "cpu": "1",
                 "ram": "256",
-                "hdd": "512M",
+                "hdd": "0.5",
```

Comment:
Phew, can we increase that to 1GB (directly in the PR or a separate one), 512 is a stupid number to calculate. The question is, how tight is that after deploying?

Reply:
Can change in separate PR